packages/react-components/react-carousel-preview/library/src/components/CarouselAutoplayButton/CarouselAutoplayButton.types.ts

```diff
@@ -14,31 +14,25 @@ export type CarouselAutoplayChangeData = EventData<'click', React.MouseEvent<HTM
   autoplay: boolean;
 };
 
+export type AutoplayAriaLabelFunction = (autoplay: boolean) => string;
+
 /**
  * CarouselAutoplayButton Props
  */
 export type CarouselAutoplayButtonProps = ToggleButtonProps &
   ComponentProps<CarouselAutoplayButtonSlots> & {
     /**
-     * Controls whether autoplay will initialized as true or false
-     * Default: true
-     */
-    defaultAutoplay?: boolean;
-
-    /**
-     * User controlled autoplay state
+     * Callback that informs the user when internal autoplay value has changed
      */
-    autoplay?: boolean;
+    onCheckedChange?: EventHandler<CarouselAutoplayChangeData>;
 
     /**
-     * Callback that informs the user when internal autoplay value has changed
+     * Override aria label property to provide state
      */
-    onAutoplayChange?: EventHandler<CarouselAutoplayChangeData>;
+    autoplayAriaLabel?: AutoplayAriaLabelFunction;
```

Comment:
I am _unsure_ that there are other components following this pattern, however there are components that have `strings` for localization:

<img width="435" alt="image" src="https://github.com/user-attachments/assets/32430cbe-df63-4472-97e8-23743b94a056">

As this one is `boolean`, should we just have `strings: { checkedAriaLabel: string; uncheckedAriaLabel }`? 🤔

Reply:
I'm a little nervous about this standard as I previously implemented it in some components but it was not well received - I think having a direct property that clearly links to aria label at the top level is a little easier to find than a strings prop nesting them (as users can just type 'aria' and IntelliSense will point them in the right direction) - but I am open to either, perhaps we can solve this as part of bug bash and make a decision about standardizing how we standardize non-prop aria strings for all of v9 so that user expectations are aligned on all components.